server/.gitignore

```diff
@@ -0,0 +1,24 @@
+# See https://help.github.com/articles/ignoring-files/ for more about ignoring files.
+
+# dependencies
+/node_modules
+/.pnp
+.pnp.js
+
+# testing
+/coverage
+
+# production
+/build
+
+# misc
+.env
+.DS_Store
+.env.local
+.env.development.local
+.env.test.local
+.env.production.local
+
+npm-debug.log*
+yarn-debug.log*
+yarn-error.log*
```

Comment:
This doesn't hurt; it allows the `server` to be even more of a standalone system. I support this! :smiley:

[nit] I'm a minimalist coder so I wonder if all of these things are needed? I realize this is a copy-paste from the `client`. Probably just `node_modules`, `.env` and (just for fun) `npm-debug.log` are needed.

Reply:
OK, I made these changes. It looks like I can just push the changes without having to do another PR, but maybe that's not right? Anyway, that's what I did...